specification/schemaregistry/data-plane/Microsoft.EventHub/preview/2021-11-01-preview/schemaregistry.json

```diff
@@ -0,0 +1,515 @@
+              },
+              "Content-Type": {
+                "description": "The serialization type for given schema. Specific type expressed in optional parameter.",
+                "type": "string"
+              },
+              "Schema-Id": {
+                "type": "string",
+                "description": "References specific schema in registry namespace."
+              },
+              "Schema-Id-Location": {
+                "type": "string",
+                "description": "URL location of schema, identified by schema ID."
+              },
+              "Schema-Group-Name": {
+                "type": "string",
+                "description": "References schema group."
+              },
+              "Schema-Name": {
+                "type": "string",
+                "description": "References schema name."
+              },
+              "Schema-Version": {
+                "type": "integer",
+                "description": "Version of the returned schema."
+              }
+            }
+          },
+          "406": {
+            "description": "Not Acceptable"
+          },
+          "default": {
+            "description": "An error response received from the Azure Schema Registry service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SchemaId": {
+      "type": "object",
+      "description": "Object received from the registry containing schema identifiers.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Schema ID that uniquely identifies a schema in the registry namespace."
+        }
+      }
+    },
+    "SchemaGroups": {
+      "type": "object",
+      "description": "Array received from the registry containing the list of schema groups.",
+      "properties": {
+        "schemaGroups": {
+          "type": "array",
+          "description": "Array of schema groups.",
+          "items": {
+            "$ref": "#/definitions/SchemaGroup"
+          }
+        }
+      }
+    },
+    "SchemaGroup": {
+      "type": "string",
+      "description": "Name of individual schema group in list of schema groups."
+    },
+    "SchemaVersions": {
+      "type": "object",
+      "description": "Array received from the registry containing the list of versions for specific schema.",
+      "properties": {
+        "schemaVersions": {
+          "type": "array",
+          "description": "Array of schema groups.",
+          "items": {
+            "$ref": "#/definitions/SchemaVersion"
+          }
+        }
+      }
+    },
+    "SchemaVersion": {
+      "type": "integer",
+      "description": "Version number of specific schema."
+    },
```

Comment:
This is one of the two "hidden APIs" that we discussed and don't want in the SDK, so it would be helpful to not change the output of this call since this will be a breaking change from the service side.